src/sage/modular/overconvergent/genus0.py

```diff
@@ -882,23 +883,6 @@ def ngens(self):
         """
         return Infinity
 
-    def gens_dict(self):
```

Comment:
Shouldn't we first deprecate this method ?

Reply:
given that it returns not implemented error, and explains itself to be meaningless I think we can just remove it